### ISSUE TITLE
[1.20.1] Exception in server tick loop when using Quarry inside claimed FTB Chunks, chunk.

### ISSUE THREAD

**Issue by Gbergz** [Kotori316/QuarryPlus#262]
## Describe the bug
Placed and used a quarry inside a claimed ftb chunks, chunk.

## To Reproduce
Steps to reproduce the behavior: (not entirely sure if this is accurate)

1. Use FTB Chunks. 
2. Claim chunk.
3. place quarry etc
crash.

## Versions

* Minecraft version: 1.20.1
* Single Player or Multi Player?: Multiplayer.
* Forge version: 47.1.0
* QuarryPlus version: 1.20.1-1201.0.9-SNAPSHOT
* Other mods: https://gist.github.com/Gbergz/00ff890e575a47446e1c9a4923402fa4

## Crash report or logs
Crash Report: https://gist.githubusercontent.com/Gbergz/702cc17b800698e8ca9fa29cdda9b0a8/raw/e54ba71e05ac8d0e2ea58b49981631dadc36e038/gistfile1.txt

**-** Gbergz

**Comment by Kotori316** [comment]
Thank you for reporting. I should update my code to use the latest version of FTB mods.

**Comment by Kotori316** [comment]
I'm sorry for this inconvenience, but I uploaded fixed version(`v1201.0.10-SNAPSHOT`) to both CurseForge and Modrinth.

I tried in my local environment in Single Player mode, and the quarry worked fine. I should note that the quarry doesn't work in claimed area. QuarryPlus and ChunkDestroyer will not work in claimed area even if they get energy from generators. The machine might leave logs to chat or server log if it detects claimed chunks.
